packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -586,7 +587,7 @@ class FloatingActionButton extends StatelessWidget {
 
     Widget result = RawMaterialButton(
       onPressed: onPressed,
-      mouseCursor: mouseCursor,
+      mouseCursor: _EffectiveMouseCursor(mouseCursor, floatingActionButtonTheme.mouseCursor),
```

Comment:
This behavior should be documented (similar to [`Slider.mouseCursor`](https://github.com/flutter/flutter/blob/a3fd36a11500643681f302455891b842be1fbb75/packages/flutter/lib/src/material/slider.dart#L376-L395))

Reply:
Do you mean something in addition to the `flutter.material.RawMaterialButton.mouseCursor` doc template that already lists pressed, hovered, focused, and disabled states?

https://github.com/flutter/flutter/blob/d2853dce6c143e998176ab5c2e2fc0b35afa0c4e/packages/flutter/lib/src/material/button.dart#L119-L130

Reply:
right! ig i missed